types/budgets.d.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+    module LH {
+        module Budgets {
+            export interface Json {
+                budgets: Array<Budget>;
```

Comment:
a tad unfortunate that for our case it will need to be `config: {budgets: {budgets: [...]}}` 

maybe we could just use the array and have the file reading piece handle the `{budgets: []}` file format or are there other sibling properties that will directly impact how we interpret the budgets?

Reply:
+1, I'm also not wild about this. However, long-term I could see that object having properties besides "budgets" (e.g. global settings about LightWallet like number of runs to use, where to log reports, etc.), so that's why it's currently this way.

Reply:
> However, long-term I could see that object having properties besides "budgets" (e.g. global settings about LightWallet like number of runs to use, where to log reports, etc.), so that's why it's currently this way.

those examples, at least, feel pretty independent of the individual budgets part. There's no reason we can't add those things as siblings on the config object rather than nested under budgets